bsdploy/bootstrap-files/pkg.conf

```diff
@@ -1 +1,8 @@
 ASSUME_ALWAYS_YES: YES
+REPO_AUTOUPDATE: NO
```

Comment:
this was more or less a drive-by-fix to pkg's annoyting default behavior. i don't want to make it an option, we're allowed to be opinionated, afterall :)